pkgs/tools/networking/fast-ssh/default.nix

```diff
@@ -18,6 +18,11 @@ rustPlatform.buildRustPackage rec {
 
   cargoSha256 = "sha256-sIQNoH3UWX3SwCFCPZEREIFR7C28ml4oGsrq6wuOAT0=";
 
+  patches = [
+    # Can be removed as soon as this is is merged: https://github.com/Julien-R44/fast-ssh/pull/22
+    ./fix-ambiguous-as_ref.patch
```

Comment:
can use fetchpatch here with your commit:
https://github.com/Julien-R44/fast-ssh/commit/c082a64a4b412380b2ab145c24161fdaa26175db.patch

Reply:
@a-n-n-a-l-e-e I could; is that okay?

I've been told elsewhere that adding `fetchpatch` statements for commits of open upstream PRs is not desirable since they could still disappear. For my own mental model: Is this different because I opened the PR myself?

Reply:
https://github.com/Julien-R44/fast-ssh/pull/22.patch is not stable nor is https://github.com/Julien-R44/fast-ssh/pull/22/commits/c082a64a4b412380b2ab145c24161fdaa26175db.patch as i think that will have problems if re-based, but i think https://github.com/Julien-R44/fast-ssh/commit/c082a64a4b412380b2ab145c24161fdaa26175db.patch is fine. do you happen to have a link to the discussion saying it is a problem?

Reply:
I didn't find it while having a quick look through my PRs here, so it was likely an offline discussion during the last ZurichZHF. Given that it's also not unlikely that I didn't understand the entire argument :)

But that "my" commit will remain accessible through the upstream repo regardless of what happens to the PR is good to know, thanks! I'll amend the commit accordingly.